### PR TITLE
Set operator image to default on engagement termination

### PR DESCRIPTION
**Pull request by rasmustautsglia**
When operator ended the engagement, the chat bubble, on the client side, was set to generic silhouette placeholder.

This fix keeps the default operator image until the user has ended the engagement on their side.

MOB-1428

### CODE REVIEW

**Review by dukhovnyi — APPROVED** [review]
Looks good 👉
Just a few things to improve in this PR
- I'd like to propose to name branch as ticket number, like `MOB-001`
- To easily keep on track of ticket progress in Jira we need to add to commit description ticket number. It can be done on last line in ticket description like this 
```
This is commit title
Here is some details about changes
MOB-001
```

**Review by dukhovnyi — COMMENTED** [review]
will we show embedded into SDK `placeholder` after ending engagement or `site default image`?

**Review by gersonnoboa — APPROVED** [review]

**Comment by gersonnoboa** [comment]
Could you rephrase your question @dukhovnyi ? So we can check that before merging

**Comment by rasmustautsglia** [comment]
!merge
